tests/CMakeLists.txt

```diff
@@ -260,8 +260,11 @@ add_test_case(rfc822_invalid_format)
 add_test_case(rfc822_invalid_tz)
 add_test_case(rfc822_invalid_auto_format)
 add_test_case(iso8601_utc_parsing)
+add_test_case(iso8601_basic_utc_parsing)
 add_test_case(iso8601_utc_parsing_auto_detect)
```

Comment:
The existence of the source/date_time.c code is pretty mysterious to me in a proof package.  It would probably help if there were documentation in the file that you are writing tests.